server.js

```diff
@@ -349,6 +349,47 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// Wercker V3 integration
```

Comment:
The old API looks like it no longer works for anyone. If you can confirm, you should remove the corresponding code, and just put "Wercker integration" in the comment.